docs/docs/api/06-Configuration/askui-ui-control-client.md

```diff
@@ -0,0 +1,80 @@
+---
+custom_edit_url: null
+---
+# askui UI Control Client
+
+## Table of Contents
+
+  - [Properties](#properties)
+    - [uiControllerUrl](#uiControllerUrl)
+    - [inferenceClientUrl](#inferenceClientUrl)
+    - [annotationLevel](#annotationLevel)
+    - [credentials](#credentials)
+  - [Set Log Level](#set-log-level)  
+  - [Example](#example)  
+## Properties
+
+### uiControllerUrl
+
+• **uiControllerUrl**: `string` - Default: `http://localhost:6769`
+
+This provides the url where the askui UI Controller runs. The askui UI Control Client is also connecting
+to this address.
+___
+
+### inferenceClientUrl
+
+• **inferenceClientUrl**: `string` - Default: `https://inference.askui.com`
+
+The address of the askui Inference server.
+___
+
+### annotationLevel
+
+• **annotationLevel**: `AnnotationLevel` - Default: `AnnotationLevel.DISABLED`
+
+AnnotationLevel is implemented as an enum. You have three options: `DISABLED`, `ON_FAILURE`, `ALL`.
+
+`ALL`:  Runs the [annotate](../../general/05-Tooling/annotate-image.md) command after the execution of each test step.
+
+
+`ON_FAILURE`: Runs the annotate command if the test step fails 
+
+
+`DISABLED`: Never runs the annotate command after test steps
+___
+
+### credentials
+
+• **credentials**: `string` - Default: `environment-tenant`, `environment-token`, `environment-email`
+
+We need to provide credentials for the authentication of the askui Inferrence Server. If you want to set your own credentials you need to provide the following arguments `tenant`, `token`, `email`. Per default we read the environment variables.
+___
+
+## Set Log Level
+
+Starting the askui library the askui UI Controller will write [logs](../06-Configuration/askui-ui-controller.md#loglevel) in a default/passed path. ([logfile path](../06-Configuration/askui-ui-controller.md#logfilepath)).
+Only the askui Client can write logs to the stdout while using the askui library.
+You can set all log levels(`"fatal"`, `"error"`, `"warn"`, `"info"`, `"debug"`, `"trace"`, `"silent"`, `"verbose"`) for the client.
```

Comment:
Does this really also set the logs of the askui UI Controller? I thought that this was only for the client lib. 

Maybe, we can improve the wording a bit. What about the following?

You can set the log level using the `LOG_LEVEL` environment variable. The following values are available:
- `"fatal"`, 
- `"error"`, 
- `"warn"`, 
- `"info"`, 
- `"debug"`, 
- `"trace"`, 
- `"silent"` and 
- `"verbose"`.

Maybe, we can also add an explanation of some log levels as at least I would not be sure of what `"silent"` actually means.

Reply:
This is not the environment variable, during the starting of the ui-controller we can select the log_levwel there, default is set to verbose as far as I remember.